testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -49,6 +49,14 @@ Map djangoConfigModels = [
     githubTeamReviewers: 'testeng'
 ]
 
+Map edxOrganizations = [
+    org: 'edx',
+    repoName: 'edx-organizations',
+    cronValue: '@weekly',
+    githubUserReviewers: 'feanil',
+    githubTeamReviewers: 'testeng'
```

Comment:
I realize that the team owner for these two repos is unclear, but we should really start figuring that out so testeng doesn't become the dumping ground for all abandoned repos.